src/cmd/linuxkit/build.go

```diff
@@ -223,7 +217,7 @@ func build(args []string) {
 		}
 
 		log.Infof("Create outputs:")
-		err = moby.Formats(filepath.Join(*buildDir, name), image, buildFormats, size, !*buildDisableTrust, cacheDir)
+		err = moby.Formats(filepath.Join(*buildDir, name), image, buildFormats, size, false, cacheDir)
```

Comment:
Why do we pass `false` to trust, as opposed to just getting rid of it as an arg entirely?